lib/l10n/intl_en.arb

```
@@ -1293,6 +1293,13 @@
   "shareFeatureComingSoon": "Share feature coming in Phase 2",
   "devTools": "Dev Tools",
   "devToolsWarning": "For debugging and troubleshooting only",
-  "viewAndExportLogs": "View and export application logs"
+  "viewAndExportLogs": "View and export application logs",
+  "saveLogs": "Save Logs",
+  "logsExportSuccess": "Logs exported successfully",
+  "logsExportError": "Failed to export logs",
+  "shareLogsError": "Failed to share logs",
+  "shareLogs": "Share Logs",
```

RuboCop (1.82.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)